app/lib/clients/vault.rb

```diff
@@ -13,7 +14,9 @@ def kv_read(path)
       end
 
       def kv_write(path, data)
-        enable_engine(kv_mount, "kv-v2")
+        unless client.sys.mounts.key?(kv_mount.to_sym)
+          enable_engine(kv_mount, "kv-v2")
```

Comment:
Can you extract the magic string here and below as you did for kv_mount?

Reply:
done